### PR TITLE
Document ltree support in EFCore.PG

### PR DESCRIPTION
**Pull request by roji**
See https://github.com/npgsql/efcore.pg/issues/1592

### CODE REVIEW

**Review by YohDeadfall — APPROVED** [review]

File: conceptual/EFCore.PG/mapping/translations.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ Regex.IsMatch(stringValue, "^A+", regexOptions)               | [stringValue ~ '
 The below translations provide functionality for determining the similarity of alphanumeric text based on trigram matching, using the [`pg_trgm`](https://www.postgresql.org/docs/current/pgtrgm.html) extension which is bundled with standard PostgreSQL distributions. All the below parameters are strings.
 
 > [!NOTE]
-> To use these translations, your project must depend on the [Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams/) package, and call <xref:Microsoft.EntityFrameworkCore.NpgsqlTrigramsDbContextOptionsBuilderExtensions.UseTrigrams%2A> in your `OnModelConfiguring`.
+> Prior to version 6.0, to use these translations, your project must depend on the [Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams/) package, and call <xref:Microsoft.EntityFrameworkCore.NpgsqlTrigramsDbContextOptionsBuilderExtensions.UseTrigrams%2A> in your `OnModelConfiguring`.
```
Comment (vonzshik) [inline conceptual/EFCore.PG/mapping/translations.md:154]: Didn't you integrate `Trigrams` package into the main one?

Comment (vonzshik) [reply conceptual/EFCore.PG/mapping/translations.md:154]: Oh wait, that's only for the previous versions, my bad 😢

**Review by vonzshik — APPROVED** [review]

File: conceptual/EFCore.PG/mapping/translations.md (hunk 1 above)
Comment (roji) [reply conceptual/EFCore.PG/mapping/translations.md:154]: Yep :)

**Comment by leonkosak** [comment]
@roji, so when .NET 6.0 (and EF Core 6.0) are released, querying hierarchy data via linq will be lik for SQL Server?
https://www.meziantou.net/using-hierarchyid-with-entity-framework-core.htm

**Comment by roji** [comment]
@leonkosak no, ltree doesn't work exactly the same way as SQL Server's hierarchy ID - it's a different feature in a different database, even if it addresses some of the same needs. I'd recommend taking a look at the [ltree documentation](https://www.postgresql.org/docs/current/ltree.html) to see what it looks like - the EFCore.PG provider pretty much exposes what you see there.

On a side note, you should be able to test the ltree support right now, by using 6.0.0-preview2.

**Comment by ronnyek** [comment]
Should this documentation include an example of this? I'm struggling to find a reference to what my entity itself should look like and how I'd simply set a path within a ltree column, let alone use the new functions to query them. Is this literally just mapping my object with a `LTree` type (instead of string etc) and then setting value with `SomeRoot.SubNode.SubNode` string?

I'd be more than happy to submit a PR to include something like that in the documentation so the initial dive doesn't feel as much of a mystery to the next guy

**Comment by roji** [comment]
@ronnyek we have the following sentence in the [LTree translations section of the docs](https://www.npgsql.org/efcore/mapping/translations.html#ltree-functions):

> Use the [LTree](https://www.npgsql.org/efcore/api/Microsoft.EntityFrameworkCore.LTree.html) type to represent ltree and invoke methods on it in EF Core LINQ queries.

We generally don't go into documenting how the PG feature works - that's the responsibility of the PG documentation (linked to from the above), and the above seems sufficient to me - can you explain what exactly you're having trouble with?

**Pull request merged**